src/lapis/nginx.conf

```diff
@@ -0,0 +1,42 @@
+worker_processes ${{NUM_WORKERS}};
+error_log stderr notice;
+daemon off;
+pid logs/nginx.pid;
+
+events {
+  worker_connections 1024;
+}
+
+# https://github.com/leafo/lapis/issues/57
+env POSTGRES_HOST;
+env POSTGRES_USER;
+env POSTGRES_PASSWORD;
+env POSTGRES_DB;
+env PORT;
```

Comment:
We have to define these values here in order for them to get passed through to the Lapis workers if we start the Lapis server.